src/Razor/test/testapps/nuget.config

```diff
@@ -1,6 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <packageSources>
-    <add key="pregenerated" value=".\PregeneratedPackages\" />
```

Comment:
?

Reply:
That folder no longer exists, I assume it was missed when a bunch of testapps were removed. I'll just delete the whole config file now that it's empty.